ajax/libs/ui.multiselect/0.3.1/js/ui.multiselect.js

```diff
@@ -0,0 +1,336 @@
+/*
+ * jQuery UI Multiselect
+ *
+ * Authors:
+ *  Michael Aufreiter (quasipartikel.at)
+ *  Yanick Rochon (yanick.rochon[at]gmail[dot]com)
+ * 
+ * Dual licensed under the MIT (MIT-LICENSE.txt)
+ * and GPL (GPL-LICENSE.txt) licenses.
```

Comment:
We might need to confirm if there is GPL license used.

Reply:
@extend1994 According to this [link](https://github.com/michael/multiselect/tree/next) and this [page](http://quasipartikel.at/multiselect_next/)
The widget under MIT and GPL v2.0 license